AndroidNativeLoginTemplate/app/src/main/java/com/salesforce/androidnativelogintemplate/NativeLogin.kt

```diff
@@ -214,7 +325,7 @@ class NativeLogin : ComponentActivity() {
      * null if one could not be obtained.  A toast will already have been shown
      * if null is returned.
      */
-    private suspend fun submitOtpDeliveryRequest(
+    private suspend fun submitStartPasswordLessLoginRequest(
```

Comment:
Here, the app demonstrates how I'm hoping the new MSDK methods are much more intuitively named.